runhouse/servers/http/http_server.py

```diff
@@ -124,6 +126,16 @@ async def ainitialize(
             logger.info(f"setting logs level to {log_level}")
         runtime_env = {"conda": conda_env} if conda_env else None
 
+        enable_telemetry_collection = kwargs.get("enable_telemetry_collection", None)
```

Comment:
do we need all this logic if we just set the default to True?

Reply:
I think we do: it covers the edge case when in the user's runhouse config `enable_telemetry_collection` is set to `true`, but he or she set restarted a specific cluster with `enable_telemetry_collection = false` (or the other way around)